token-metadata/program/src/state.rs

```diff
@@ -0,0 +1,38 @@
+use {
+    borsh::{BorshDeserialize, BorshSerialize},
+    solana_program::pubkey::Pubkey,
+};
+/// prefix used for PDAs to avoid certain collision attacks
+pub const PREFIX: &str = "metadata";
+
+pub const NAME_LENGTH: usize = 32;
+
+pub const SYMBOL_LENGTH: usize = 10;
+
+pub const URI_LENGTH: usize = 200;
+
+pub const METADATA_LEN: usize = 32 + NAME_LENGTH + SYMBOL_LENGTH + URI_LENGTH + 200;
+
+pub const OWNER_LEN: usize = 32 + 32 + 200;
```

Comment:
Why +200?   But also it seems like we should be able to derive this length from the struct